README.md

```diff
@@ -47,7 +47,8 @@ KubeEdge consists of cloud part and edge part.
 - [CloudHub](https://kubeedge.io/en/docs/architecture/cloud/cloudhub): a web socket server responsible for watching changes at the cloud side, caching and sending messages to EdgeHub.
 - [EdgeController](https://kubeedge.io/en/docs/architecture/cloud/edge_controller): an extended kubernetes controller which manages edge nodes and pods metadata so that the data can be targeted to a specific edge node.
 - [DeviceController](https://kubeedge.io/en/docs/architecture/cloud/device_controller): an extended kubernetes controller which manages devices so that the device metadata/status data can be synced between edge and cloud.
-
+- SyncController: periodically check data that persists on edge and cloud, trigger reconcile if necessary.
```

Comment:
meta data

Reply:
an extended kubernetes controller which is responsible for reliable data transmission between cloud and edge. It periodically check meta data that persists on edge and cloud, trigger reconcile if necessary.